aion/lib/aion/room_channel/room.ex

```diff
@@ -76,12 +89,24 @@ defmodule Aion.RoomChannel.Room do
     room.question
   end
 
-  @spec get_new_question_with_answers(integer) :: %{question: Question.t, answers: list Answer.t}
-  defp get_new_question_with_answers(room_id) do
-      question = Question.get_random_question(room_id)
-      answers = Answer.get_answers(question.id)
-      Logger.debug fn -> "Answers: #{inspect(Enum.map(answers, fn answer -> answer.content end))}" end
+  @spec get_new_question_with_answers(question_list, integer) ::  new_question_set
+  defp get_new_question_with_answers(questions, room_id) do
```

Comment:
This name doesn't sound intuitive. You're calling it from `change_question` and it actually changes the current question, right? Perhaps there will be a need for another function apart from change_question, but this is not the case. You can safely move it to `change_question`.